der/src/asn1/any.rs

```diff
@@ -23,6 +23,7 @@ use crate::asn1::ObjectIdentifier;
 /// Nevertheless, this crate defines an `ANY` type as it remains a familiar
 /// and useful concept which is still extensively used in things like
 /// PKI-related RFCs.
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
```

Comment:
```suggestion
#[cfg_attr(
    feature = "arbitrary",
    derive(arbitrary::Arbitrary),
    allow(clippy::integer_arithmetic)
)]
```

Reply:
I tried this and a few other places for the `allow` but none suppresses the warning. I don't know how to put attributes on derive generated code and adding the `allow` to the module seems a bit extreme (suppressing more than wanted).

Reply:
I guess the issue is it's annotating the struct and not the associated `impl` blocks.

You can add it at to the module if it's gated on `feature = "arbitrary"`